README.md

```diff
@@ -318,10 +318,9 @@ Automatically infer schema (data types)
 ```R
 library(SparkR)
 
-Sys.setenv('SPARKR_SUBMIT_ARGS'='"--packages" "com.databricks:spark-xml_2.10:0.5.0" "sparkr-shell"')
-sqlContext <- sparkRSQL.init(sc)
+sparkR.session("local[4]", sparkPackages = c("com.databricks:spark-xml_2.10:0.5"))
 
-df <- read.df(sqlContext, "books.xml", source = "xml", rowTag = "book")
+df <- read.df("books.xml", source = "xml", rowTag = "book")
```

Comment:
BTW I changed this to "xml" from the full package name, and I admit I didn't test it. I assume SparkR can specify these the same way.

Reply:
Ah, I quickly tested. Both should be fine ..